plugins/postcss-trigonometric-functions/test/basic.expect.css

```diff
@@ -2,7 +2,7 @@
 	order: 0.70711; /* 45deg */
 	order: 0.70711; /* 45deg */
 	order: 0.70711; /* 45deg */
-	order: 0.70711; /* 45deg */
+	order: sin(1 / 8turn); /* 45deg */
```

Comment:
I think this is invalid.
But `1turn / 8` would not be invalid.

I can update the source file.